utils/python/CIME/XML/entry_id.py

```diff
@@ -34,6 +34,17 @@ def get_default_value(self, node, attributes={}):
             value = ""
         return value
 
+    def set_default_value(self, vid, val, attributes={}):
```

Comment:
If a job is submitted with --no-batch and RESUBMIT > 0 as in the ERR test we need to make sure that the --no-batch is respected for all of the jobs.   So I set the default_value to the value for that system and update the value to "none"